public/static/docs/install/pre-release.md

```diff
@@ -15,7 +15,7 @@ $ pip install git+https://github.com/iterative/dvc
 ```
 
 > `gitpython` allows the installation process to generate a DVC version using
-> the current Git commit SHA. This lets us to distinguish official DVC releases
+> the current Git commit hash. This lets us to distinguish official DVC releases
```

Comment:
we can keep both - `SHA hash`

Reply:
Will review these (per https://github.com/iterative/dvc.org/pull/933#pullrequestreview-348856746)